### PR TITLE
fixing docker registry component tag search

### PR DESCRIPTION
**Pull request by adauto**
* This pull request aims to fix the bug of component's image tag search on Villager and Moove modules

### CODE REVIEW

File: ui/src/core/components/Form/Input/__tests__/Input.spec.tsx [hunk 1]
```diff
@@ -67,3 +68,29 @@ test('floating label when Input has value', () => {
...
+  const { getByTestId } = render(<Input name="keyName" label="Label" isLoading />);
+  const loading = getByTestId('icon-ellipse-loading');
+
+  expect(loading).toBeInTheDocument();
+});
+
+test('renders Input component error', async () => {
+  const { container, getByTestId } = render(
+    <Input
+      isError
```
Comment (washingtonsoares) [inline ui/src/core/components/Form/Input/__tests__/Input.spec.tsx:82]:
```suggestion
      hasError
```

File: villager/src/test/java/io/charlescd/villager/test/GetDockerRegistryTagHandlerTest.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
+package io.charlescd.villager.test;
```
Comment (ifdouglas) [inline villager/src/test/java/io/charlescd/villager/test/GetDockerRegistryTagHandlerTest.java:1]: The LICENSE header is missing

Comment (adauto) [reply villager/src/test/java/io/charlescd/villager/test/GetDockerRegistryTagHandlerTest.java:17]: Corrigido

File: ui/src/core/components/Form/Input/__tests__/Input.spec.tsx (hunk 1 above)
Comment (Leandroqo) [reply ui/src/core/components/Form/Input/__tests__/Input.spec.tsx]: done

**Pull request merged**